core/src/main/java/jenkins/tools/ToolConfigurationCategory.java

```diff
@@ -0,0 +1,21 @@
+package jenkins.tools;
```

Comment:
BTW this is actually not usable from plugins, they just have to implement `ToolInstallation` and declare a `ToolDescriptor`.